.gulp/regeneration.iced

```diff
@@ -54,6 +54,7 @@ regenExpected = (opts,done) ->
         return done()
 
 goMappings = {
+  'additional-properties':['additionalProperties.json', 'additionalproperties'],
```

Comment:
is there anything other tests we are missing, if so maybe open an issue for it

Reply:
Right now we have a generic issue tracking this, https://github.com/Azure/azure-sdk-for-go/issues/1428

Reply:
eh ok, but we dont get a test coverage report anymore right , and sorry for my grammar, sleepy

Reply:
You can find the coverage report here http://azure.github.io/autorest/dashboard.html

Reply:
oh didnt know about it. can we make version 3 not show on top ? :D